ui/widgets/nuxeo-directory-radio-group.js

```diff
@@ -150,7 +150,13 @@ import { DirectoryWidgetBehavior } from './nuxeo-directory-widget-behavior.js';
     }
 
     _isChecked(entry) {
-      const value = (this.value && this.value.length > 0 ? this.value : null) || this.selectedItem;
+      let value;
+      if ((this.value && this.value.length > 0) || typeof this.object === 'object') {
+        [value] = this;
+      } else if (this.selectedItem) {
```

Comment:
If we just did an `else` here we would have the same result, right?
```suggestion
      } else {
```
We could also make this smaller by initializing value with `this.selectedItem`.